contents/customers/rayfit.md

```diff
@@ -0,0 +1,66 @@
+
+For a team deep in the technical trenches of machine learning and computer vision, PostHog's "one-stop shop" setup meaningfully reduced the time and effort of building out their stack. [Analytics](/docs/product-analytics), [feature flags](/docs/feature-flags), and a [data warehouse](/docs/data-warehouse) could all live one in place.
+
+That solved the infrastructure problem. Next was ensuring the entire team was on board.
+
+## Making data accessible with PostHog AI
+
+For Rayfit, the early friction wasn’t collecting data, it was making sure everyone could access it. 
+
+"PostHog can present as really technical," Alan explains. “Our CEO and designer had used Mixpanel and Amplitude before — which felt more approachable since PostHog is built for engineers.”
+
+Like many, their in-app data was also tricky to access. It requires [SQL skills](/product-engineers/sql-for-analytics) not everyone has. "A lot of the data we use for the generated workouts lives in the app itself," Alan says. "It wouldn't make sense to send a unique event for each one. With the warehouse, we can join it to a person and learn more about their behaviors."
+
+That’s where [PostHog AI](/ai) comes in. For Alan, SQL is second nature — AI just removes some friction. For the rest of the team, having AI draft and edit queries makes the whole workflow far more accessible. “PostHog AI has actually been a deciding factor in our decision to stick with PostHog,” he told us. “Without it, we probably would have moved to a different analytics package because the learning curve felt so steep.”
+
+<OSQuote
+  customer="rayfit"
+  author="alan_yang"
+  quote={0}
+/>
+
+## Realistic expectations, real results
+
+As an early adopter of AI products (in and outside of PostHog), Alan is pragmatic about AI capabilities. “All LLMs right now are about 30–50%,” he says. “But it’s a nice problem space because that’s still a win when it gets it right.”
+
+So when PostHog AI doesn’t nail the syntax on the first try, he doesn’t see it as a failure — just part of the workflow. In fact, it mirrors how both Rayfit and PostHog operate. “We're shipping things before they're fully baked just like PostHog,” he says. “So when AI features don't work perfectly, everyone is willing to try again.”
+
+Even 30–50% accuracy is a win when the alternative is teammates getting stuck or pinging Alan for every question. “It’s saved me countless hours because the data we have is so much more self-serve,” he says. “Anyone on the team can ask questions to PostHog AI.”
+
+What surprised him most is how quickly others leaned into it. “Some of the engineers are using PostHog AI way more than me,” Alan admits. “They're building insights and dashboards, and each of them has workflows I never really thought about.”
+
+![rayfit moving average chart](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/rayfit_rep_counting_chart_5a99280ef4.png)
```

Comment:
Could we use this screen grab instead? Has a better title for external readers

<img width="765" height="720" alt="Image" src="https://github.com/user-attachments/assets/a7f281d4-2c32-456e-8949-148db5bda239" />